resources/views/includes/structure/email-signup.blade.php

```diff
@@ -1,18 +1,21 @@
-<div class="container-fluid justify-content-center bg-white p-3">
-    <div class="col-md-12 mx-auto">
-        <div class="col-md-6 mx-auto">
-            <div class="btn-sign rounded-left rounded-right">
-                <div class="btn-sign-box"></div>
-                <div class="btn-sign-text">
-                    <h5>
-                        <a href="https://tickets.museums.cam.ac.uk/account/create" class="stretch-link">
-                            Sign up for updates
-                        </a>
-                    </h5>
-                    <p class="text-center">Updates about future exhibitions and displays, family activities, virtual
-                        events & news. You'll be the first to know...</p>
-                </div>
-            </div>
+<div class="bg-white bg-white-subscribe">
+    <div class="container container-subscribre">
+        <div class="wrapper">
+            <h5>Sign up to our emails</h5>
+            <p>Be the first to hear about our news, exhibitions, events and more…</p>
         </div>
+        <a class="subscribre-button" href="https://tickets.museums.cam.ac.uk/account/create">
+            <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" width="24" height="25" viewBox="0 0 24 25" fill="none">
```

Comment:
Hey @ostafinskim rather than inline the SVG, why not use the blade helper syntax and use a font awesome icon using code like the below (not sure what the icon shortcut is though off the top of my head) with integer height and width:
```
@svg('fas-envelope',['width'=> x, 'height' => y])
``` 
All the icons get cached for speed.

Reply:
Unfortunately I couldn't find free fontawesome svg that matches to current desing.